extensions/groovy-actions/karaf-kar/pom.xml

```diff
@@ -61,7 +69,11 @@
             <artifactId>jsoup</artifactId>
             <version>1.13.1</version>
         </dependency>
-
+        <dependency>
+            <groupId>com.sun.activation</groupId>
```

Comment:
I added it because of this error message: org.apache.felix.resolver.reason.ReasonException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=unomi-groovy-actions; type=karaf.feature; version="[1.6.0.SNAPSHOT,1.6.0.SNAPSHOT]"; filter:="(&(osgi.identity=unomi-groovy-actions)(type=karaf.feature)(version>=1.6.0.SNAPSHOT)(version<=1.6.0.SNAPSHOT))" [caused by: Unable to resolve unomi-groovy-actions/1.6.0.SNAPSHOT: missing requirement [unomi-groovy-actions/1.6.0.SNAPSHOT] osgi.identity; osgi.identity=org.apache.unomi.groovy-actions-rest; type=osgi.bundle; version="[1.6.0.SNAPSHOT,1.6.0.SNAPSHOT]"; resolution:=mandatory [caused by: Unable to resolve org.apache.unomi.groovy-actions-rest/1.6.0.SNAPSHOT: missing requirement [org.apache.unomi.groovy-actions-rest/1.6.0.SNAPSHOT] osgi.wiring.package; filter:="(&(osgi.wiring.package=javax.activation)(version>=1.2.0)(!(version>=2.0.0)))"]]